internal/buildscripts/packaging/installer/install.sh

```diff
@@ -599,10 +586,42 @@ EOH
   fi
 }
 
+create_zeroconfig_dotnet() {
+  local otlp_endpoint="$1"
+  local version="$( get_package_version splunk-otel-auto-instrumentation )"
+  local resource_attributes="splunk.zc.method=splunk-otel-auto-instrumentation-${version}"
+
+  if [ -n "$deployment_environment" ]; then
+    resource_attributes="${resource_attributes},deployment.environment=${deployment_environment}"
+  fi
+
+  backup_file "$dotnet_zeroconfig_path"
+
+  echo "Creating ${dotnet_zeroconfig_path}"
+  cat <<EOH > $dotnet_zeroconfig_path
+CORECLR_ENABLE_PROFILING=1
+CORECLR_PROFILER={918728DD-259F-4A6A-AC2B-B85E1B658318}
+CORECLR_PROFILER_PATH=${dotnet_install_dir}/linux-x64/OpenTelemetry.AutoInstrumentation.Native.so
```

Comment:
I guess this script doesn't support arm nor Alpine, is that correct?

Reply:
> I guess this script doesn't support arm nor Alpine, is that correct?

Correct.  The installer script only supports deb/rpm distros, and `splunk-otel-dotnet` does not currently support arm64.